app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "SempoBlockchain",
-  "version": "1.2.7",
+  "version": "1.2.10",
```

Comment:
skips some verisons?

Reply:
Lol I based it off another fix branch, will fix for merge.